lebedigital/BeamDesign/BeamDesign.py

```diff
@@ -0,0 +1,166 @@
+# -*- coding: utf-8 -*-
+"""
+Created on Fri Sep 16 11:13:33 2022
+
+@author: zv1174
+"""
+import math
+
+def section_dimension_rule_of_thumb(span:float)->tuple:
+    """
+    This is as per eurocode guideline choice of section dimension based on span 
+
+    Parameters
+    ----------
+    span : float
+        beam span.
+
+    Returns
+    -------
+    tuple
+        tuple of breath and depth of the beam.
```

Comment:
what is breath? height?

Reply:
i will make it consistent width and depth